tests/test_testPipeline.py

```diff
@@ -121,6 +121,7 @@ def setUpClass(cls):
         butlerTests.addDatasetType(cls.repo, "visitSsObjects", cls.visitOnlyId.keys(), "DataFrame")
         butlerTests.addDatasetType(cls.repo, "apdb_marker", cls.visitId.keys(), "Config")
         butlerTests.addDatasetType(cls.repo, "deepDiff_associDiaSrc", cls.visitId.keys(), "DataFrame")
+        butlerTests.addDatasetType(cls.repo, "deepDiff_longTrailedSrc", cls.visitId.keys(), "DataFrame")
```

Comment:
(If you decide to go with `longTrailedSources` instead of `Src`, you'll need to also change it here)

Reply:
WAIT, is the line above this a typo? It says `associDiaSrc` and I am 99% sure it should be `assocDiaSrc`, what is happening here?